docs/DevGuide/Notation.md

```diff
@@ -0,0 +1,139 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Notation Used by SpECTRE {#spectre_notation_general}
+
+# Current Notation {#spectre_current_notation}
+
+\tableofcontents
+
+## Discontinuous Galerkin Notation {#dg_notation}
+
+- The logical coordinates in the element are
+  \f$\boldsymbol{\xi}=\{\xi,\eta,\zeta\}\f$.
+- The Jacobian matrix is defined as
+  \f{align*}{
+      \mathbf{J}=\frac{\partial \mathbf{x}}{\partial \boldsymbol{\xi}}
+  \f}
+  where \f$\mathbf{x}\f$ are the coordinates mapped to by the coordinate map.
+  The determinant of the Jacobian matrix is denoted by \f$J\f$.
+- We use \f$s\f$ and \f$t\f$ to be grid point indices. That is, \f$U_s\f$ is
+  \f$U\f$ at the grid point \f$s\f$. For tensor product bases we index the
+  individual irreducible topologies by subscripting \f$s\f$. For example,
+  \f$s_1\f$ would be the \f$\xi\f$-dimension in the reference element.
+- We use a nodal basis of Lagrange interpolating polynomials
+  \f{align*}{
+      \ell_{s_i}(\xi)=\prod^{N}_{\substack{t_i=0\\ t_i\ne s_i}}
+      \frac{\xi - \xi_{t_i}}{\xi_{s_i}-\xi_{t_i}},
+  \f}
+  where \f$N\f$ is the order/degree of the basis in each logical direction.
+- A DG scheme with an \f$N^{\mathrm{th}}\f$ order basis is referred to as a
+  \f$P_N\f$ scheme and converges at order \f$\mathcal{O}\left(\Delta
+  x^{N+1}\right)\f$ where \f$\Delta x\f$ is the 1d size of the element.
```

Comment:
I think "converges at order O(Dx^{N+1})" is unclear as to whether it is the error term or the last correct term that is proportional to Dx^{N+1}.